uswds/uswds-site
# USWDS-Site: Remove check-contrast script and its dependencies

## Description

Removes unused script because security issues were appearing in its dependencies. Closes #1511.

## Additional information

Removed 5 dependencies
- chroma-js
- colors
- json-format
- minimist
- yamljs

## How to test
This was a standalone script that wasn't called anywhere. It shouldn't have any effect on the site.

---

Before you hit Submit, make sure you’ve done whichever of these applies to you:
